calculateMetrics/metrics.py

```diff
@@ -0,0 +1,381 @@
+            commits_sd * 1.0
+        ) and github_data[username_accesser][1] >= np.average(commits_list) - (
+            commits_sd * 1.0
+        ):
+            print("appending 4 to commit_scores list")
+            commit_scores.append(4)
+            username_accesser += 1
+        elif github_data[username_accesser][1] <= np.average(commits_list) + (
+            commits_sd * 1.5
+        ) and github_data[username_accesser][1] >= np.average(commits_list) - (
+            commits_sd * 1.5
+        ):
+            print("appending 3 to commit_scores list")
+            commit_scores.append(3)
+            username_accesser += 1
+        elif github_data[username_accesser][1] <= np.average(commits_list) + (
+            commits_sd * 2.0
+        ) and github_data[username_accesser][1] >= np.average(commits_list) - (
+            commits_sd * 2.0
+        ):
+            print("appending 2 to commit_scores list")
+            commit_scores.append(2)
+            username_accesser += 1
+        elif github_data[username_accesser][1] <= np.average(commits_list) + (
+            commits_sd * 2.25
+        ) and github_data[username_accesser][1] >= np.average(commits_list) - (
+            commits_sd * 2.25
+        ):
+            print("appending 1 to commit_scores list")
+            commit_scores.append(1)
+            username_accesser += 1
+        else:
+            print("Appending 0 to commit_scores list")
+            commit_scores.append(0)
+            username_accesser += 1
+    global commits_overall_score
+    commits_overall_score = np.average(commit_scores)
+    # These print statments print out the scores for the amount of commits the team
+    # added
+    print("")
+    print("Each users scores are listed below: ")
+    print(commit_scores)
+    print("The average of these scores is: ", np.average(commit_scores))
+    print(
+        "The team earned a score of: [",
+        np.average(commit_scores),
+        "/ 5] for teamwork on commits.",
+    )
+    print("")
+    return commit_scores
+
+
+def added_calculator():
+    """This will determine how well the team worked together by analyzing the spread of lines of code added"""
+    global standard_deviations_list
+    global added_list
+    print("Lines added average is: ", np.average(added_list))
+    print("Lines added standard deviation is: ", np.std(added_list))
+    # Standard deviation of lines added
+    added_sd = standard_deviations_list[1]
+    added_scores = []
+    username_accesser = 0
+    list_length = len(github_data)
+    # for ab in range(len(github_data)):
+    for cd in range(len(github_data)):
+        print("Checking GitHub user: ", github_data[username_accesser][0])
+        # This if/else calculates the amount added to the standard deviation,
+        # in order to grade the team.
+        # This if statment will end if the user was within 0.5 of the standard deviation.
+        if github_data[username_accesser][2] <= np.average(added_list) + (
```

Comment:
Since `github_data[username_accesser][2]` and `np.average(added_list)` are being used all over the place, do you think if it would be a good idea to store their values into variables and just call the variables?